### PR TITLE
Adding clear queue fn

### PR DESCRIPTION
**Pull request by pirumpi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,6 +34,11 @@ function queue() {
     // Call the command
     command();
   };
+  //Clearing quote
+  quote.clear = function(){
```
Comment (johnnyman727) [inline index.js]: Did you mean `queue.clear`?

**Comment by johnnyman727** [comment]
@pirumpi just curious, why are you adding this function?

**Comment by pirumpi** [comment]
I'm creating a TTS api using emic2 and I want to provide the option to stop all the commands in the queue. Right now I can stop the current playing text to speech message but I cannot stop any queue messages in the stack. This function will allow me to enhance the stop function.

**Comment by johnnyman727** [comment]
Ah gotcha. Well go ahead and fix the one issue I commented on and I'm happy to merge it in. 

-Jon

**Comment by pirumpi** [comment]
Done

**Pull request merged**